GLFW-b.cabal

```diff
@@ -16,14 +16,14 @@ description:
   library for creating windows with OpenGL contexts and managing input and
   events.
   .
-  GLFW-b depends on bindings-GLFW
+  This version of GLFW-b depends on git branch Upgrade-3.3 of bindings-GLFW
```

Comment:
I would suggest we start work on upgrading `bindings-GLFW` first before changing this text.